openmetadata-ui/src/main/resources/ui/src/components/common/SuccessScreen/SuccessScreen.tsx

```diff
@@ -104,7 +104,7 @@ const SuccessScreen = ({
           <Typography.Paragraph
             className="m-b-0"
             data-testid="success-line"
-            ellipsis={{ rows: 3 }}>
+            ellipsis={{ rows: 7 }}>
```

Comment:
We can remove this no? Why we want to show ellipsis when we intent to show full error or success

Reply:
Good point, I'll remove it.